appeals/web/src/server/lib/mappers/mapper-types-jsdoc.js

```diff
@@ -330,4 +330,5 @@
  * @property {number} [body.title.headingLevel]
  * @property {string} [body.title.classes]
  * @property {PageBodyRow[]} [body.rows]
+ * @property {PageComponent[]} [pageComponents]
```

Comment:
Good shout on this - in future we could probably simplify the confirmation pages a lot by using `PageComponent` exclusively